fecgov/fec-cms
# Embedded audit search

Embedded audit search block to be used in Wagtail  streamfield to add it to  these two pages:
https://www.fec.gov/legal-resources/enforcement/audit-reports
https://www.fec.gov/legal-resources/enforcement/

- Resolves https://github.com/fecgov/fec-cms/issues/1360
Note: Using a templatetag to make this streamfield block has the added benefit of making it available to add to hardcoded pages as well as from within the Wagtail editing interface. 

### [How to test &#187;](#how-to-test)

- [ ]  Completion criteria includes adding this to the two wagtail pages above in production

### Impacted areas of the application
modified:   home/blocks.py
new file:     home/templates/blocks/audit.html
new file:     home/templates/partials/audit-search.html
modified:   home/templatetags/audit_search.py

### Screenshots:
<img width="1215" alt="screen shot 2018-07-01 at 10 02 09 pm" src="https://user-images.githubusercontent.com/5572856/42142001-e89b5c92-7d7a-11e8-83b5-867329b98e3a.png">
<hr>

![steramblock](https://user-images.githubusercontent.com/5572856/42111594-f64283d0-7bb2-11e8-8478-9011b2a38a35.gif)
<a id="how-to-test"></a> 
<hr/>

**How to Test:**
1) Checkout and run this branch locally: `feature/embedded-audit-search `
2) Point to some instance of Open FEC that populates datatables locally
3) Go to the wagtail admin and edit `audit-reports` http://127.0.0.1:8000/admin/pages/6660/edit/
4) Under sections, add a new content section (the plus symbol). Choose Audit reports block from the choices.(See GIF above)
5) Preview or publish the page and see the embedded audit search.
6) Test the embedded audit search locally
